egs/wsj/s5/local/wsj_word_class_prep.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright 2010-2016 FAU Erlangen  (Author: Axel Horndasch).  Apache 2.0.
+
+# To be run from one directory above this script.
+#
+# the main purpose of the script is to copy / convert word class files
+# from an external source (${CORPUS_WORD_CLASS_DIR}) to the place in
+# the Kaldi directory structure (${KALDI_WORD_CLASS_DIR})
+
+export LC_ALL=C
+
+CORPUS_NAME=wsj
+
+if [ $# != 2 ]; then
+  echo "Usage: ./local/${CORPUS_NAME}_word_class_prep.sh /path/to/corpus/word/class/files new/kaldi/word/class/dir"
```

Comment:
The script (which is now called egs/wsj/s5local/wsj_wclass_prep.sh) has been revised as well and so have the perl scripts you mentioned.